packages/csharp/PortingAssistant/PortingAssistant.Api/Application.cs

```diff
@@ -117,8 +117,10 @@ public void SetupConnection(bool console = false)
                 var httpService = _services.GetRequiredService<IHttpService>();
                 try
                 {
-                    var fileToDownload = "newtonsoft.json.json.gz";
-                    using Stream stream = httpService.DownloadS3FileAsync(fileToDownload).Result;
+                    var file1 = httpService.DownloadS3FileAsync("newtonsoft.json.json.gz");
```

Comment:
Why you have two same functionalities in both c# and electron side. We could just keep one